app/src/components/Modal.vue

```diff
@@ -0,0 +1,47 @@
+<template>
```

Comment:
In a future PR, this component will display detailed information for a selected Record. For now, I'm leaving its skeleton here.